matlab/CARFAC_SAI_hacking.m

```diff
@@ -19,8 +19,15 @@
 %% Test/demo hacking for CARFAC_SAI Matlab stuff:
 
 clear variables
+clear_old_files = true;
+
+[status, msg, msgID] = mkdir('frames');
+
+if (status && clear_old_files && strcmp(msgID, 'MATLAB:MKDIR:DirectoryExists'))
+    disp("Clearing old frames");
+    delete('frames/*');
+end
```

Comment:
This changes the behavior of previous code: the previous code always deletes all the images but this one won't delete the images after the computation. Maybe let's move the deletion to the end of the computation?